tasks/createPosition/index.ts

```diff
@@ -25,10 +25,6 @@ task('createPosition', 'Create stETH position on AAVE')
   .addOptionalParam('multiply', 'Required multiply', 2, types.float)
   .addFlag('dummyswap', 'Use dummy swap')
   .setAction(async (taskArgs, hre) => {
-    if (!process.env.SERVICE_REGISTRY_ADDRESS) {
-      throw new Error('SERVICE_REGISTRY_ADDRESS env variable is not set')
```

Comment:
Why did you remove this?

Reply:
We have it in parameteres: 
`  .addOptionalParam<string>('serviceRegistry', 'Service Registry address')`
So we don't need to have it in env

Reply:
Well I understand but as you can see the param is optional so I might not pass it and since we won't have a place to read the ServiceRegistry from  the ask will fail. The param is created ( my assumption ) just to override the `.env` configuration if the developer needs that for some reason.
Also even if you rename `addOptionalParam` to just `addPAram` that way every single time i have to go around and look for the serviceRegistry address somewhere.

Reply:
I will move it after reading it from env or params